cmd/taskhandler/main.go

```diff
@@ -52,9 +51,12 @@ func serveCache() func() error {
 func serveProxy() {
 
 	var (
-		metricsPath = viper.GetString("metrics.metricsPath")
-		restPort    = viper.GetInt("proxyRestPort")
-		grpcPort    = viper.GetInt("proxyGrpcPort")
+		restPort = viper.GetInt("proxyRestPort")
+		restHost = viper.GetString("serving.restHost")
+		grpcPort = viper.GetInt("proxyGrpcPort")
+
+		metricsPath    = viper.GetString("metrics.path")
+		metricsTimeout = viper.GetInt("metrics.timeout")
```

Comment:
I think we should add a config named `serving.metricsPath` in case the serving metric path is different from the other metrics path to support cases in which it is not feasible to change the serving metrics path.. If not set, we can default to `metrics.path`.

Reply:
I agree, done